bciers/apps/reporting/src/app/components/finalReview/FinalReviewForm.tsx

```diff
@@ -66,13 +58,87 @@ export const FinalReviewForm: React.FC<Props> = ({
               key={facilityKey}
               facilityName={facilityReport.facility_name}
               facilityData={facilityReport}
-              showReportingOnlyConditions={!isReportingOnly()}
+              showReportingOnlyConditions={!isReportingOnly}
```

Comment:
I'm confused by this part (and maybe it's just the naming that's confusing). But if the operation is reporting-only, I would expect `isReportingOnly` to be true, and therefore I would expect `showReportingOnlyConditions` to be sent a value of `true`, not the inverse. Should the `!` be there?